devel/external/AEGIS/AEGIS_25.2_training_sample_readme.txt

```diff
@@ -0,0 +1,109 @@
+This README file describes the AEGIS F814W<25.2 training sample, which is a
+randomly ordered sample of galaxies in the AEGIS field with AB magnitude<25.2, in F606W(V) and F814W(I) bands.
+
+NOTE: The columns in the catalog are those expected by galsim.RealGalaxyCatalog.
+However, some parts of that analysis, i.e parametric fits, were not performed.
+Hence those values were manually set and do not have physical meaning.
```

Comment:
I'm a little surprised by this comment.  The main ```real_galaxy_catalog_SAMPLE.fits``` file format used for galsim.RealGalaxyCatalog() doesn't have any information about parametric fits, so it should be possible to define all columns in a way that is meaningful for this sample.  Which ones were you not able to do?